Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -700,6 +700,29 @@ public extension UIView {
         }
         return views
     }
+    
+    /// SwifterSwift: Adds a horizontal separator to the bottom of the view.
+    ///
+    ///     let view = UIView()
+    ///     view.addBottomSeparator(color: .lightGray, height: 1, spacing: 8)
+    ///
+    /// - Parameters:
+    ///   - color: The separator color. Default is `.black`.
+    ///   - height: The height of the separator. Default is `1`.
+    ///   - spacing: Spacing from the bottom edge. Default is `0`.
+    func addBottomSeparator(color: UIColor = .black, height: CGFloat = 1, spacing: CGFloat = 0) {
+        let line = UIView()
+        line.translatesAutoresizingMaskIntoConstraints = false
+        line.backgroundColor = color
+        line.layer.cornerRadius = height / 2
+        self.addSubview(line)
+        NSLayoutConstraint.activate([
+            line.heightAnchor.constraint(equalToConstant: height),
+            line.leadingAnchor.constraint(equalTo: self.leadingAnchor),
+            line.trailingAnchor.constraint(equalTo: self.trailingAnchor),
```

Comment:
I don't think you need `self.` here. `swiftformat` should've taken care of that. Do you have it installed?
```suggestion
            line.leadingAnchor.constraint(equalTo: leadingAnchor),
            line.trailingAnchor.constraint(equalTo: trailingAnchor),
```